### PR TITLE
xandzia: add task_11

### PR DESCRIPTION
**Pull request by xandzia**
test pull request

task_11

- [demo](https://xandzia.github.io/kottans_frontend/task_11/index.html)

- [repo](https://github.com/xandzia/kottans_frontend/tree/master/task_11)

### CODE REVIEW

**Review by viktor-yakubiv — CHANGES_REQUESTED** [review]
You must not commit generated files, like cache ones, to the repository.

Please, review your global `.gitignore` or add local one.

---

Do not commit unnecessary files. It harms people who read your code. Just find the file, see it's empty and probably is not used and wondering, why is it here?

File: task_11/index.html [hunk 1]
```diff
@@ -0,0 +1,249 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <!--    Define keywords for search engines:-->
```
Comment (viktor-yakubiv) [inline task_11/index.html]: Code must be self-documented as much as possible.

Highly suggest you avoid stupid comments for lines which are clear without it.

**Comment by viktor-yakubiv** [comment]
Also, pay attention to indentation and line limitations up 80, 100 or 120 characters long. Follow clear convention of code writing.

**Comment by viktor-yakubiv** [comment]
If you use a tool, e.g. SASS, use it wisely.

In this case, do not create a mess in the repository. Now you have `styles.scss` in the root, which contains css-reset, three files in the `sass` directory: two are empty, one contains reset too. But you could have one clean directory with sass-modules, actually well-named files.

Think about [project structure](http://thesassway.com/beginner/how-to-structure-a-sass-project).

**Comment by viktor-yakubiv** [comment]
All active element __must__ respond to user interaction. It means, any button, link, radio or checkbox must have hover and focus styles.

**Review by viktor-yakubiv — CHANGES_REQUESTED** [review]
All active element must respond to user interaction. It means, any button, link, radio or checkbox must have hover and focus styles.

**Review by viktor-yakubiv — APPROVED** [review]

**Comment by viktor-yakubiv** [comment]
It's much better! Good job!

**Comment by viktor-yakubiv** [comment]
You just deleted too many files, but it's okay. You may leave only one pretty file with styles.

**Comment by viktor-yakubiv** [comment]
@AMashoshyna, are you able to review?

**Review by feroxes — APPROVED** [review]

**Pull request merged**